tests/Feature/RemoteWorkers.php

```diff
@@ -17,6 +17,7 @@ protected function setUp(): void
         URL::forceRootUrl('http://localhost');
         Config::set('cdash.remote_workers', true);
         Config::set('cdash.backup_timeframe', 0);
+        Config::set('filesystem.default', 'local');
```

Comment:
If we're always going to run this against the local disk, should we just disable the test via CTest instead?